android/assets/jsons/Tutorials.json

```diff
@@ -192,16 +193,16 @@
         "When founding a city, it won't follow a religion immediately.",
         "The religion a city follows depends on the total pressure each religion has within the city.",
         "Followers are allocated in the same proportions as these pressures, and these followers can be viewed in the city screen.",
-        "Based on this, you can get a feel for which religions have a lot of pressure build up in the city, and which have almost none.",
-        "The city follows a religion if a majority of its population follows the city, and will only than be effected by Follower and Pantheon beliefs of that religion.",
+        "Based on this, you can get a feel for which religions have a lot of pressure built up in the city, and which have almost none.",
+        "The city follows a religion if a majority of its population follows that religion, and will only then receive the effects of Follower and Pantheon beliefs of that religion.",
         "",
         "Spreading religion happens naturally, but can be sped up using missionaries or great prophets.",
-        "Missionaries can be bought in cities following a major religion, and take the religion of that city.",
-        "So do take care where you are buying them! If another civilization has converted one of your cities to their religion, missionaries bought their will follow their religion.",
-        "Great prophets always have your religion, regardless if they are bought or spawned naturally, but captured great prophets do retain their original religion.",
-        "Both great prophets and missionaries are able to spread religion to cities when they are inside their borders, even cities of other civilizations.",
+        "Missionaries can be bought in cities following a major religion, and will take the religion of that city.",
+        "So do take care where you are buying them! If another civilization has converted one of your cities to their religion, missionaries bought there will follow their religion.",
+        "Great prophets always have your religion when they appear, even if they are bought in cities follow other religions, but captured great prophets do retain their original religion.",
```

Comment:
'in cities follow other religions' from my point of view, it should be 'in cities following other religions' or 'in cities that follows other religions'